packages/compass-global-writes/src/components/states/shard-key-correct.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useMemo } from 'react';
+import {
+  Banner,
+  BannerVariant,
+  Body,
+  css,
+  Link,
+  spacing,
+  Code,
+  Subtitle,
+  Label,
+  SpinLoader,
+  Button,
+} from '@mongodb-js/compass-components';
+import { connect } from 'react-redux';
+import {
+  ShardingStatuses,
+  unmanageNamespace,
+  type RootState,
+  type ShardKey,
+  type ShardZoneData,
+} from '../../store/reducer';
+import toNS from 'mongodb-ns';
+import { ShardZonesTable } from '../shard-zones-table';
+import { useConnectionInfo } from '@mongodb-js/compass-connections/provider';
+
+const nbsp = '\u00a0';
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[400],
+});
+
+const codeBlockContainerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[100],
+  maxWidth: '700px',
+});
+
+const paragraphStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[100],
+});
+
+export type ShardKeyCorrectProps = {
+  namespace: string;
+  shardKey: ShardKey;
+  shardZones: ShardZoneData[];
+  isUnmanagingNamespace: boolean;
+  onUnmanageNamespace: () => void;
+};
+
+export function ShardKeyCorrect({
```

Comment:
Would be nice to have a bit of padding at the bottom here
<img width="356" alt="image" src="https://github.com/user-attachments/assets/1653b737-d43b-44d3-b1fe-0cd9c08e25a9">